compiler/ir/ir.tree/src/org/jetbrains/kotlin/ir/declarations/impl/IrFieldImpl.kt

```diff
@@ -53,10 +53,20 @@ class IrFieldImpl(
         origin: IrDeclarationOrigin,
         symbol: IrFieldSymbol,
         type: IrType
+    ) :
+            this(startOffset, endOffset, origin, symbol, type, symbol.descriptor.visibility)
+
+    constructor(
+        startOffset: Int,
+        endOffset: Int,
+        origin: IrDeclarationOrigin,
+        symbol: IrFieldSymbol,
+        type: IrType,
+        visibility: Visibility
```

Comment:
I'd prefer to add a default parameter to an existing constructor

Reply:
Done